x-pack/legacy/plugins/fleet/scripts/readme.md

```diff
@@ -1,3 +1,18 @@
+### Dev agents
+
+you can run a development fleet agent that is going to enroll and checkin every 3 seconds.
+For this you can run the following command in the fleet pluging directory.
+
+```
+npx ts-node scripts/dev_agent.ts <enrollmentToken> --kibanaUrl=http://localhost:5603/qed
```

Comment:
Rather than using `ts-node` please use a `.js` file that requires [`src/setup_node_env`](https://github.com/elastic/kibana/tree/master/src/setup_node_env) then requires a `.ts` file, like all our other scripts: https://github.com/elastic/kibana/blob/master/scripts/build_sass.js#L20-L21